apps/web/pages/booking/[uid].tsx

```diff
@@ -574,15 +573,40 @@ export default function Success(props: SuccessProps) {
                             )}
                           </>
                         );
-                      })}
-                    {bookingInfo?.smsReminderNumber && hasSMSAttendeeAction && (
+                      })} */}
+                    {/* {bookingInfo?.smsReminderNumber && hasSMSAttendeeAction && (
                       <>
                         <div className="mt-9 font-medium">{t("number_sms_notifications")}</div>
                         <div className="col-span-2 mb-2 mt-9">
                           <p>{bookingInfo.smsReminderNumber}</p>
                         </div>
                       </>
-                    )}
+                    )} */}
+                    {Object.entries(bookingInfo.responses).map(([name, response]) => {
+                      const field = eventType.bookingFields.find((field) => field.name === name);
+                      // We show location in the "where" section
+                      // We show Booker Name, Emails and guests in Who section
+                      // We show notes in additional notes section
+                      if (
+                        !field ||
+                        field.name === "location" ||
+                        field.name === "guests" ||
+                        field.name === "notes" ||
+                        field.name === "name" ||
+                        field.name === "email"
+                      ) {
+                        return null;
+                      }
```

Comment:
https://handbook.cal.com/engineering/best-practices/avoid-comparing-multiple-values-with-includes

```suggestion
                      if (!field || ["location", "guests", "notes", "name", "email"].includes(field.name)) {
                        return null;
                      }
```